app/src/ui-blokada/res/values/strings_panel.xml

```diff
@@ -37,7 +37,7 @@
     <!-- How many rules are configured. Parameters is number of rules. -->
     <string name="panel_ruleset_title">Filters now have %s rules.</string>
     <string name="panel_ruleset_updating">Updating filters...</string>
-    <string name="panel_ruleset_built">Filters are now configured with &lt;b&gt;%s&lt;/b&gt; rules. Up to &lt;b&gt;%s&lt;/b&gt; rules can fit into your device memory overall. Remember, this is just an estimate and subject to change depending on other apps on your device, so you should stay well below this limit to be safe.</string>
+    <string name="panel_ruleset_built">Filters are now configured with &lt;b&gt;%s&lt;/b&gt; rules. Your device allows max &lt;b&gt;%s&lt;/b&gt; MB of memory for every application, and up to &lt;b&gt;%s&lt;/b&gt; rules can fit into this allocated value.&lt;br&gt;Remember, this is just an estimate and Blokada utilises the most memory when it merges the lists.</string>
```

Comment:
When you change the number of placeholders in a string, please introduce a new string (change its name), or there might be problems with old translations.

Reply:
Maybe change "allocated value" with "it", this sounds too scary

Reply:
Ok, I'll keep it in mind. Fixed in a new commit.